packages/maitreya-deb/maitreya-deb.pacscript

```diff
@@ -1,8 +1,7 @@
-name="maitreya8-deb"
+name="maitreya-deb"
 repology=("project: maitreya")
-pkgname="maitreya8"
 version="8.0.1b"
-url="https://github.com/martin-pe/${pkgname}/releases/download/v${version}/${pkgname}_${version}_amd64.deb"
+url="https://github.com/martin-pe/maitreya8/releases/download/v${version}/maitreya8_${version}_amd64.deb"
 depends=("libwxsqlite3-3.0-0" "wx-common wx3.0-i18n")
 replace=("maitreya8")
```

Comment:
```suggestion
replace=("maitreya")
```

Reply:
I kept both names actually because the deb from github actually installs as maitreya8